js/sync-meta-box/index.js

```diff
@@ -1,4 +1,18 @@
 import draftLiveDiffBtnHandler from './diff-view';
+
+const commentButton = jQuery('#comment-button');
+const input = jQuery('#comment-input');
+
+commentButton.off('click').on('click', function() {
+
```

Comment:
Att denna inte funkar ibland kan vara orsakat av att just input är definerat globalt.
Definera input när du ska använda den så du vet att du får samma referens till elementet.
```suggestion
const input = jQuery('#comment-input');
```